### ISSUE TITLE
Suggestions Menu not closing after selection

### ISSUE THREAD

**Issue by larsvombruch** [maheshmnj/searchfield#10]
The suggestions do not close after having tapped on them, this is my code: 

```
SearchField(
                    suggestions: [
                      "London",
                      "Paris",
                      "Kemer",
                      "Berlin",
                    ],
                    controller: departureController,
                    onTap: (value) {},
                    hasOverlay: false,
                    validator: (p0) {},
                    searchInputDecoration: InputDecoration(
                      labelText: "Abfertiger",
                      floatingLabelBehavior: FloatingLabelBehavior.always,
                      labelStyle: TextStyle(fontSize: 20, color: Colors.black),
                      border: OutlineInputBorder(
                        borderSide: BorderSide(
                          width: 2,
                          color: Colors.grey,
                        ),
                      ),
                      focusedBorder: OutlineInputBorder(
                        borderSide: BorderSide(
                          width: 2,
                          color: Colors.greenAccent,
                        ),
                      ),
                    ),
                  ),
```

**Comment by larsvombruch** [comment]
The suggestions do not close after having tapped on them, this is my code: 

```
SearchField(
                    suggestions: [
                      "London",
                      "Paris",
                      "Kemer",
                      "Berlin",
                    ],
                    controller: departureController,
                    onTap: (value) {},
                    hasOverlay: false,
                    validator: (p0) {},
                    searchInputDecoration: InputDecoration(
                      labelText: "Abfertiger",
                      floatingLabelBehavior: FloatingLabelBehavior.always,
                      labelStyle: TextStyle(fontSize: 20, color: Colors.black),
                      border: OutlineInputBorder(
                        borderSide: BorderSide(
                          width: 2,
                          color: Colors.grey,
                        ),
                      ),
                      focusedBorder: OutlineInputBorder(
                        borderSide: BorderSide(
                          width: 2,
                          color: Colors.greenAccent,
                        ),
                      ),
                    ),
                  ),
```

**Comment by larsvombruch** [comment]
Also, is there a way to show the suggestions right from the start, or do they always appear only after the first input? If so, is there a way to change it?

**Comment by maheshmnj** [comment]
Hi @larsvombruch, I could not reproduce your issue please share the output of `flutter doctor -v` and the version of searchfield you are using.

> Also, is there a way to show the suggestions right from the start, or do they always appear only after the first input? If so, is there a way to change it?

For this a new version has been published this is available in version 0.5.4

**Comment by larsvombruch** [comment]
Thank you very much, that seems to work. Is there a way to specifiy a textInputAction like in normal TextFields that shifts to the next Field or something similar? Creating and handling lifecycles of many FocusNodes manually can be tedious, since I am using a stateless widget for my form. Any plans on implementing that, or should I come up with a solution and create a PR for that?

**Comment by maheshmnj** [comment]
Great, let's talk about it in another issue. Please file a new issue with more details 
- problem
- proposal/solution 

Closing this issue as solved.If you disagree feel free to write in the comments and I will reopen it.

Thank you.
